libs/application/templates/parental-leave/src/forms/EditOrAddPeriods.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  buildCustomField,
+  buildDateField,
+  buildForm,
+  buildMultiField,
+  buildRepeater,
+  buildSection,
+  buildSelectField,
+  buildSubmitField,
+  Form,
+  FormModes,
+} from '@island.is/application/core'
+import Logo from '../assets/Logo'
+import { parentalLeaveFormMessages } from '../lib/messages'
+
+export const EditOrAddPeriods: Form = buildForm({
+  id: 'ParentalLeaveEditOrAddPeriods',
+  title: 'Edit or add periods',
+  logo: Logo,
+  mode: FormModes.EDITING,
+  children: [
+    buildSection({
+      id: 'editOrAddPeropds',
+      title: '',
+      children: [
+        buildRepeater({
+          id: 'periods',
+          title: parentalLeaveFormMessages.leavePlan.title,
+          component: 'PeriodsRepeater',
+          children: [
+            buildDateField({
+              id: 'startDate',
+              title: parentalLeaveFormMessages.startDate.title,
+              description: parentalLeaveFormMessages.startDate.description,
+              placeholder: parentalLeaveFormMessages.startDate.placeholder,
+            }),
+            buildMultiField({
+              id: 'endDate',
+              title: parentalLeaveFormMessages.endDate.title,
+              description: parentalLeaveFormMessages.endDate.description,
+              children: [
+                buildDateField({
+                  id: 'endDate',
+                  title: parentalLeaveFormMessages.endDate.label,
+                  placeholder: parentalLeaveFormMessages.endDate.placeholder,
+                }),
+              ],
+            }),
+            // buildCustomField(
+            //   {
+            //     id: 'endDate',
+            //     name: m.duration,
+            //     description: parentalLeaveFormMessages.duration.description,
+            //     component: 'ParentalLeaveDuration',
+            //   },
+            //   {
+            //     showTimeline: true,
+            //   },
+            // ),
```

Comment:
Can this be removed?

Reply:
Yeah I'm not sure why we have this (it is also in ParentalLeaveForm.ts) I've removed it here.